### PR TITLE
Feature/co 64 co 65

### PR DESCRIPTION
**Pull request by DanielVelando**
Still needs a little styling/finish here and there - but for now I think we should leave it as it is, until the last few outstanding have been clarified?

### CODE REVIEW

File: frontend/src/components/CarInfo/CarInfoComp.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+          </FormControl>
+          <FormControl isInvalid={formSubmitAttempts > 0 && !localCarInfoForm.fuelType} isRequired>
+            <FormLabel>Fuel type:</FormLabel>
+            <Select
+              placeholder="Select fuel type"
+              onChange={e => updateLocalForm(e.target.value, "fuelType")}>
+              <option>Oil</option>
+              <option>Petroleum</option>
+              <option>Gasoline</option>
+              <option>Other</option>
```
Comment (objmj-itminds) [inline frontend/src/components/CarInfo/CarInfoComp.tsx:142]: You could use the enum type from nswagts

**Review by objmj-itminds — COMMENTED** [review]
As a whole, this looks good - but for now, we will park this PR.

File: frontend/src/components/CarInfo/CarInfoComp.tsx (hunk 1 above)
Comment (DanielVelando) [reply frontend/src/components/CarInfo/CarInfoComp.tsx]: Right - updated!

File: frontend/src/components/CarInfo/CarInfoComp.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+          </FormControl>
+          <FormControl isInvalid={formSubmitAttempts > 0 && !localCarInfoForm.fuelType} isRequired>
+            <FormLabel>Fuel type:</FormLabel>
+            <Select
+              placeholder="Select fuel type"
+              onChange={e => updateLocalForm(e.target.value, "fuelType")}>
+              <option>{capitalize(FuelType[FuelType.OIL])}</option>
+              <option>{capitalize(FuelType[FuelType.PETROLEUM])}</option>
+              <option>{capitalize(FuelType[FuelType.GASOLINE])}</option>
+              <option>{capitalize(FuelType[FuelType.OTHER])}</option>
```
Comment (objmj-itminds) [inline frontend/src/components/CarInfo/CarInfoComp.tsx:144]: Iteration over the enums would have been better. But leave this for now - we will return to it once we have had our feature requirement clarification.

**Comment by objmj-itminds** [comment]
Closing this as the scope has changed.

**Pull request closed without merging**